plugin_manager.py

```diff
@@ -2073,7 +2072,7 @@ async def draw_ui(self):
                        size=(0, 0),
                        h_align='center',
                        v_align='center',
-                       text=f'API Version: {babase.app.api_version}',
+                       text=f'API Version: {babase.app.api_version if build_number < 21282 else babase.app.env.api_version}',
```

Comment:
this'll introduce lots of code changes that'll become legacy code in a few months (adding on to maintainence burden), let's add a compatibility layer instead, something like this:

```python
import babase
from baenv import TARGET_BALLISTICA_BUILD

if TARGET_BALLISTICA_BUILD < 21282:
    babase.app.env.api_version = babase.app.api_version
    babase.app.env.debug_build = babase.app.api_version.debug_build
    

# Now let's use `babase.app.env.*` everywhere else
# in code irrespective of whatever build version user is on.
```
then once a couple months have passed and most ppl are on newer versions, we can then simply nuke this compatibility layer code all at once easily.